.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, "unverified"]
```

we don't have an unverified tag in this repo so idk what it'd do
```suggestion
labels: ["bug"]
```